### ISSUE TITLE
Extension issue

### ISSUE THREAD

**Issue by chrisheckler** [aws/aws-toolkit-vscode#1041]
- Issue Type: `Bug`
- Extension Name: `aws-toolkit-vscode`
- Extension Version: `1.8.0`
- OS Version: `Darwin x64 18.6.0`
- VSCode version: `1.41.1`

```json 
{
	"messages": [],
	"activationTimes": {
		"codeLoadingTime": 1001,
		"activateCallTime": 1,
		"activateResolvedTime": 2832,
		"activationReason": {
			"startup": false,
			"extensionId": {
				"value": "amazonwebservices.aws-toolkit-vscode",
				"_lower": "amazonwebservices.aws-toolkit-vscode"
			},
			"activationEvent": "onView:aws.cdk.explorer"
		}
	},
	"runtimeErrors": [
		{
			"name": "Error",
			"message": "The language client requires VS Code version ^1.42.0 but received version 1.41.1"
		}
	]
}
```

**Comment by bryceitoc9** [comment]
Version 1.8.0 of the toolkit includes LSP support for Step Functions: https://github.com/aws/aws-toolkit-vscode/blob/master/.changes/1.8.0.json#L23 ; the LSP package it's using is requiring VS Code 1.42.

Looks like this was an oversight on our part for not bumping our [package.json](https://github.com/aws/aws-toolkit-vscode/blob/master/package.json#L13)'s required VS Code version. We'll go ahead and bump that for the next release; in the meanwhile, could you either upgrade VS Code or downgrade the toolkit?

**Comment by awschristou** [comment]
This issue has been resolved with v1.9.0 of the toolkit, which is now available.
